<launch>
    <arg name="input"       default="input"/>
    <arg name="output"      default="$(arg input)/filtered"/>
    <arg name="min_range"   default="0"/>
    <arg name="max_range"   default="1"/>

    <node pkg="pointcloud_tools" type="pointcloud_range_filter" name="range_filter" output="screen">
        <remap from="~input"    to="$(arg input)"/>
        <remap from="~output"   to="$(arg input)/filtered"/>
        <param name="min_range" value="$(arg min_range)"/>
        <param name="max_range" value="$(arg max_range)"/>
    </node>
</launch>
